data/src/test/java/org/apache/iceberg/TestMergingMetrics.java

```diff
@@ -30,13 +30,14 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableList;
```

Comment:
this is the wrong import and should stay `org.apache.iceberg.relocated.com.google.common.collect....`